### PR TITLE
WSTEAMA-1407: Add script switch link to the Uzbek homepage

### PR DESCRIPTION
**Pull request by karinathomasbbc**
Resolves JIRA https://jira.dev.bbc.co.uk/browse/WSTEAMA-1407

Overall changes
======
Enables the script switch link on Uzbek homepages

Code changes
======
- Render script switch link if page type is Article or Home page
- Add unit tests

Testing
======
Unit Tests
http://localhost:7080/uzbek/lat?renderer_env=test
http://localhost:7080/uzbek/cyr?renderer_env=test

Screenshot 2024-10-16 at 17 36 19
Screenshot 2024-10-16 at 17 36 27

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

### CODE REVIEW

File: src/app/legacy/containers/Header/index.test.jsx [hunk 1]
```diff
@@ -134,7 +135,7 @@ describe(`Header`, () => {
       expect(container.querySelectorAll(scriptLinkSelector).length).toBe(1);
     });
 
-    it('should not render script link for uzbek when it is not an article page ', () => {
+    it('should not render script link for uzbek when it is not a supported page type', () => {
```
Comment (karinathomasbbc) [inline src/app/legacy/containers/Header/index.test.jsx]: Add all other page types to array & use `it.each`

**Review by amoore108 — APPROVED** [review]

**Comment by Nabeel1276** [comment]
Can I ask where you got all the new json data from for `data/uzbek/homePage/cyr.json` and `data/uzbek/homePage/lat.json`

**Comment by karinathomasbbc** [comment]
> Can I ask where you got all the new json data from for `data/uzbek/homePage/cyr.json` and `data/uzbek/homePage/lat.json`

Good question! From the Fabl API 😄

**Review by holchris — APPROVED** [review]

**Review by pvaliani — APPROVED** [review]
Niceeee 🚢

**Pull request merged**